plugins/db-mongo/src/static/health/health.service.base.ts

```diff
@@ -0,0 +1,17 @@
+import { Injectable } from "@nestjs/common";
+import { PrismaService } from "../../prisma/prisma.service";
+
+@Injectable()
+export class HealthServiceBase {
+  constructor(protected readonly prisma: PrismaService) {}
+  async isDbReady(): Promise<boolean> {
+    try {
+      await this.prisma.$runCommandRaw({
+        select: 1 
```

Comment:
@morhag90  Is this the way to that in mongo?
@yuval-hazaz this is different than what you sent me 
`await this.prisma.user.findFirst();`

Reply:
@abrl91 This line is not good because we don't know for sure that we will have a user collection